pkg/backup/pvc_snapshot_tracker.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package backup
+
+import (
+	"fmt"
+
+	corev1api "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// pvcSnapshotTracker keeps track of persistent volume claims that have been snapshotted
+// with restic.
+type pvcSnapshotTracker struct {
```

Comment:
Thanks for the comment on this. On a second pass, I'm wondering if having `restic` in the name might be helpful.

For example, in [backup.go, L289](https://github.com/heptio/ark/pull/608/files#diff-ca7997ad3f41210fc3bfc1fb6f537110R289), it's near `snapshotService`, which is for cloud volumes. Looking at the two names could get confusing, given they're for different types of snapshots.

I don't want to encode too much information in names, but the closeness here seems a bit off.

Reply:
I renamed the variable to `resticSnapshotTracker` and left the type name as `pvcSnapshotTracker`. I think that makes it pretty clear. let me know what you think.